anadi/__main__.py

```diff
@@ -8,6 +8,7 @@
 
 parser = argparse.ArgumentParser(prog="AnadiCSV", description="Handle CVS file using SQL language")
 parser.add_argument("-d", "--dir", type=str, help="CSV file dir", required=True)
```

Comment:
this one is not required anymore BUT we need to check if it has a value if -q and -f are not valueated

Reply:
Now either the dir or the file and query need to be passed.